From a7f0c7f7e10f97bd1d9737a7572d539e55eb7597 Mon Sep 17 00:00:00 2001
From: Ding Wei <leo.ding@rock-chips.com>
Date: Tue, 14 Apr 2020 17:35:26 +0800
Subject: [PATCH] video: rockchip: mpp: add session extra info for debug

current->pid may be the same for different sessions in one thread.

Change-Id: Iac6ee4c641d63cbd3e883861d339eb2d540d3118
Signed-off-by: Ding Wei <leo.ding@rock-chips.com>
---
 drivers/video/rockchip/mpp/mpp_common.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/video/rockchip/mpp/mpp_common.c b/drivers/video/rockchip/mpp/mpp_common.c
index 000596c64d0d..7316a7866bef 100644
--- a/drivers/video/rockchip/mpp/mpp_common.c
+++ b/drivers/video/rockchip/mpp/mpp_common.c
@@ -1529,8 +1529,8 @@ int mpp_time_diff(struct mpp_task *task)
 	struct mpp_dev *mpp = task->session->mpp;
 
 	do_gettimeofday(&end);
-	mpp_debug(DEBUG_TIMING, "%s: pid:%d time: %ld us\n",
-		  dev_name(mpp->dev), task->session->pid,
+	mpp_debug(DEBUG_TIMING, "%s: pid: %d, session: %p, time: %ld us\n",
+		  dev_name(mpp->dev), task->session->pid, task->session,
 		  (end.tv_sec  - task->start.tv_sec)  * 1000000 +
 		  (end.tv_usec - task->start.tv_usec));
 
-- 
2.35.3

